code_schemes/age.json

```diff
@@ -0,0 +1,1151 @@
+{
+  "SchemeID": "Scheme-4189e74c",
+  "Name": "Age",
+  "Version": "0.0.0.5",
+  "Codes": [
+    {
+      "CodeID": "code-E-720cdb66",
+      "CodeType": "Meta",
+      "MetaCode": "escalate",
+      "DisplayText": "meta: escalate",
```

Comment:
```      "DisplayText": "meta: ESCALATE",```

Please

Reply:
Will change, but worth noting that this breaks with display text formatting conventions, where we have historically only used upper case to signal control codes.

Reply:
yup - understood - intentional in this case

Reply:
(This is actually kinda control code-ish - a complex action will be taken in response, channel though to Nook and sending a pager alarm. After this project is over, let's think about how to design the formatting conventions around the increasingly complex response surface in P1/KK hybrid projects)